setup.py

```diff
@@ -5,15 +5,21 @@
 import platform
 import sys
 from os import getenv
+import os.path
 from subprocess import call
 
 import setuptools.command.build_py
+import setuptools.command.develop
+import setuptools.command.install
+
 from setuptools import find_packages, setup
 from setuptools.command.test import test as TestCommand
 
 from coalib import VERSION, assert_supported_version, get_version
+from coalib import get_current_commit, COMMIT_FILE, COALIB_DIR
 from coalib.misc.BuildManPage import BuildManPage
 
+
```

Comment:
no issue, just a new commit

I know this is trivial, but for the sake of making commits as atomic as possible and ease review (even a bit) we should do it :)